apps/expo-go/src/components/SnacksListItem.tsx

```diff
@@ -24,43 +27,60 @@ function normalizeDescription(description?: string): string | undefined {
  * the snacks list page for an account.
  */
 
-export function SnacksListItem({ description, isDraft, name, url, first, last }: Props) {
+export function SnacksListItem(snackData: Props) {
+  const { fullName, name, description, isDraft, first, last, sdkVersion } = snackData;
   const theme = useExpoTheme();
 
-  const handlePressProject = () => {
-    Linking.openURL(UrlUtils.normalizeUrl(url));
-  };
+  const normalizedDescription = normalizeDescription(description);
+  const isSupported = sdkVersion ? sdkVersion === Environment.supportedSdksString : true;
 
-  const handleLongPressProject = () => {
-    const message = UrlUtils.normalizeUrl(url);
-    Share.share({
-      title: name,
-      message,
-      url: message,
-    });
+  const handlePressProject = () => {
+    if (isSupported) {
+      Linking.openURL(UrlUtils.normalizeSnackUrl(fullName));
+    } else {
+      const expoGoMajorVersion = Environment.supportedSdksString?.split('.')[0];
+      const snackMajorVersion = sdkVersion?.split('.')[0];
+      if (expoGoMajorVersion && snackMajorVersion) {
+        Alert.alert(
+          `Selected Snack uses unsupported SDK (${snackMajorVersion})`,
+          `The currently running version of Expo Go supports SDK ${expoGoMajorVersion} only. Update your Snack to this version to run it.`
+        );
+      } else {
+        // Unlikely to hit this it's a good fallback case if we somehow get
+        // invalid data from the Snack or environment
+        Alert.alert(
+          `Selected Snack uses unsupported SDK`,
+          `Update your Snack to a compatible version to run it.`
+        );
+      }
+    }
   };
 
-  const normalizedDescription = normalizeDescription(description);
-
   return (
     <View
       border="default"
       roundedTop={first ? 'large' : undefined}
       roundedBottom={last ? 'large' : undefined}
       overflow="hidden"
-      style={{
-        borderBottomWidth: last ? 1 : 0,
-        borderTopWidth: first ? 1 : 0,
-      }}>
-      <TouchableOpacity onPress={handlePressProject} onLongPress={handleLongPressProject}>
+      style={[
+        {
+          borderBottomWidth: last ? 1 : 0,
+          borderTopWidth: first ? 1 : 0,
+        },
+      ]}>
+      <TouchableOpacity onPress={handlePressProject}>
         <View
           padding="medium"
           bg="default"
           roundedTop={first ? 'large' : undefined}
           roundedBottom={last ? 'large' : undefined}>
           <Row align="center" justify="between">
             <View align="start" flex="1">
-              <Text type="InterSemiBold" ellipsizeMode="tail" numberOfLines={1}>
+              <Text
+                type="InterSemiBold"
+                ellipsizeMode="tail"
+                numberOfLines={1}
+                style={{ opacity: isSupported ? 1 : 0.5 }}>
```

Comment:
i tried making the entire row lower opacity but it felt like a bit much, the text + the badge communicates sufficiently imo